.github/workflows/pnp-input-api-build.yml

```diff
@@ -75,9 +75,36 @@ on:
         type: boolean
         required: false
         default: false
+      dotnet-version:
+        description: ".NET version"
```

Comment:
What is the reason why do you switch to double quotes usage?

Reply:
This change was done by the yml formatter extension I use in VS Code. Are the problems with this change? Maybe you have maintainability concerns?

Reply:
Both types of quotes can be used in YAML, however I think I saw only single quotes usage in the github actions documentation, that's why I asked what was the reason for switching to double quotes

Reply:
I have found this issue https://github.com/actions/runner/issues/866 and seems double quotes are not supported in expression language in yml files. But only there.